source/docs/being-a-team/how-to-ask-effective-questions.rst

```diff
@@ -0,0 +1,95 @@
+.. include:: <isonum.txt>
+
+How to Ask Effective Questions
+==============================
+
+Asking questions effectively is one of the most important skills to develop, especially for your life outside of robotics.
+Being able to learn from your questions properly is crucial and as such, you should be able to ask good questions, help those who want to help you, and make the most of what you are taught.
```

Comment:
```suggestion
Asking questions effectively is one of the most important skills to develop, especially for your life outside of robotics. Being able to learn from your questions properly is crucial and as such, you should be able to ask good questions, help those who want to help you, and make the most of what you are taught.
```

We write paragraphs as one line, because that makes git diffs nicer. This is an example, it should be done for every paragraph.